microsoft-support.md

```diff
@@ -0,0 +1,37 @@
+# Microsoft support for grpc-dotnet
```

Comment:
Where is this doc going to be linked from?

Reply:
Have we decided whether we want to link this from the dot.net site? The only downside I see from having this information on GitHub is that it won't be localized for international customers. Right now, our site only has Japanese and Simplified Chinese, but we hope to expand that.

Reply:
There aren't plans to at the moment.